apps/website/src/app/members/page.tsx

```diff
@@ -0,0 +1,119 @@
+'use client'
+
+import React, {ButtonHTMLAttributes, useEffect, useState} from 'react'
+import Link from "next/link";
+
+
+export default function Members() {
+    const [id, getID] = useState('')
+    const [name, setName] = useState('')
+    const [email, getEmail] = useState('')
+    const rows = [];
+    const columns = [];
+    const [password, getPassword] = useState('')
+    const [buttonDisabled, setButtonDisabled] = useState(true)
+    const [userData, setUserData] = useState([])
+
+
+    useEffect (() =>{
+        const fetchData = async () => {
+            const response = await fetch('https://resident-management.fly.dev/usersList')
+            const data = await response.json()
+            if (data.status == "ok") {
+                setUserData(data.users);
+                console.log(data.users);
+            } else {
+
+            }
+        }
+        fetchData();
+    }, [])
+
+
+    async function checkAdmin(){
+        if (localStorage.getItem('usertype') === 'Admin') {
+            setButtonDisabled(false);
+            console.log("Admin Priveledges found");
+            return true;
+        }
+        else{
+            console.log("Not an Admin");
+            return false
+        }
+    }
+
+
+    async function deleteUser(name: string) {
+        setButtonDisabled(true)
+
+        const response = await fetch('https://resident-management.fly.dev/deleteUser', {
+            method: 'DELETE',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                name,
+            }),
+        })
+
+        if (await checkAdmin() || 1){
+            setButtonDisabled(true)
```

Comment:
this button disabled stuff is fine for our project, but just an fyi that you can inspect element and set `disabled: false` on the button to delete a user without admin, the check should be done in the server code

Reply:
Good to know, thanks!